app/routes/admin/permissions/index.js

```diff
@@ -0,0 +1,9 @@
+import Ember from 'ember';
+
+const { Route } = Ember;
+
+export default Route.extend({
+  beforeModel() {
+    return this.replaceWith('admin.permissions.system-roles');
```

Comment:
Please use `this.transitionTo` here. Check this https://guides.emberjs.com/v2.11.0/routing/redirection/